pyramid/config/views.py

```diff
@@ -762,6 +782,17 @@ def view(context, request):
         if context is None:
             context = for_
 
+        if exception is not None:
+            if context is not None:
+                raise ConfigurationError('view "context" and "exception" '
+                                         'arguments are mutually exclusive')
+            context = exception
+
+        isexc = isexception(context)
+        if not isexc and exception is not None:
+            raise ConfigurationError(
+                'view "exception" must be an exception type')
```

Comment:
I think that would be a good idea. I considered making `pyramid.exceptions.isexception` a public API as well. What do you think?